src/steamship/plugin/blockifier/transcriber.py

```diff
@@ -0,0 +1,77 @@
+from abc import abstractmethod
+from typing import List, Optional, Set
+
+from steamship import Block, File, SteamshipError, Tag, Task, TaskState
+from steamship.base.mime_types import MimeTypes
+from steamship.invocable import InvocableResponse
+from steamship.invocable.plugin_service import PluginRequest
+from steamship.plugin.blockifier.blockifier import Blockifier
```

Comment:
is there any pythonic way to avoid the stutter of `.blockifier.blockifier` ?   maybe we could rename the parent to something like `.content` or `.importer` or `.converter` or something?

Reply:
The stutter only applies to this file though. Since I included both Blockifier and Transcriber in the __init__ you can import Transcriber and Blockifier from the steamship.plugin.blockifier namespace (without needing the additional .blockifier).